apps/chatbot/lint-staged.config.js

```diff
@@ -1,12 +1,9 @@
-import mainConfig from "../../lint-staged.config.js";
-
 /** @param {string[]} filenames */
-const buildEslintCommand = (filenames) =>
-  `pnpm eslint --fix ${filenames.join(" ")}`;
+const buildBiomeCommand = (filenames) =>
+  `biome check --apply ${filenames.join(" ")}`;
 
 const config = {
-  ...mainConfig,
```

Comment:
Why can't we continue to inherit the main config here still, given we're using biome everywhere?